firmware/steering/kia_soul_ps/steering_control_module.ino

```diff
@@ -541,13 +467,6 @@ void loop()
 
     check_rx_timeouts();
 
-    // Read and parse incoming serial commands
-    if ( Serial.available() > 0 ) 
-	{
-        uint8_t incomingSerialByte = Serial.read();
-        processSerialByte( incomingSerialByte );
-    }
-
```

Comment:
It was decided that this was messy and confusing for users. If they want to debug using serial then they are free to add it, but otherwise we though it best to remove it. (@evanlivingston @petebrink and I. )